yt_geotiff/io.py

```diff
@@ -23,6 +25,13 @@ def _transform_data(self, data):
             data = np.flip(data, axis=self.ds._flip_axes)
         return data
 
+    def _resample(self, data, fname, scale_factor, original_res, load_res, order):
+        print('Resampling {}: {} to {} m'.format(fname,\
+         str(original_res), str(load_res)))
```

Comment:
Let's use the yt logger, which will allow the user to disable printing if they want.
```suggestion
        mylog.info(f"Resampling {fname}: {original_res} to {load_res} m.")
```

You'll need this import:
```
from yt.funcs import mylog
```